### PR TITLE
[ISSUE#5253] Fix missing identity header error

### LINKED ISSUE #5253: WARN Invalid server identity value for xxx from xxx
**Describe the bug**
我安装nacos1.4.1集群后，并添加了如下参数：
After I installed the nacos1.4.1 cluster, I added the following parameters:
```
nacos.core.auth.enabled=true
nacos.core.auth.enable.userAgentAuthWhite=false
nacos.core.auth.server.identity.key=xing
nacos.core.auth.server.identity.value=xing
```

然后，在没有任务其他条件干扰的情况下，直接去看日志core-auth.log,发现一直在报：
Then, without interference from other conditions of the task, I went directly to the log core-auth.log and found that it has been reporting:

```
2021-03-30 00:33:23,532 WARN Invalid server identity value for xing from 172.18.0.4
2021-03-30 00:33:24,652 WARN Invalid server identity value for xing from 172.18.0.3
2021-03-30 00:33:27,542 WARN Invalid server identity value for xing from 172.18.0.4
2021-03-30 00:33:28,662 WARN Invalid server identity value for xing from 172.18.0.3
2021-03-30 00:33:31,551 WARN Invalid server identity value for xing from 172.18.0.4
2021-03-30 00:33:32,671 WARN Invalid server identity value for xing from 172.18.0.3

大家有人遇过吗？
Has anyone encountered this problem?

### PR DESCRIPTION
**Pull request by cutiechi**
Please do not create a Pull Request without creating an issue first.

## What is the purpose of the change

Fix  #5253 

## Brief changelog

XX

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

* [ ] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
* [ ] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
* [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
* [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/alibaba/nacos/tree/master/test).
* [ ] Run `mvn -B clean package apache-rat:check findbugs:findbugs -Dmaven.test.skip=true` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.

### CODE REVIEW

File: core/src/main/java/com/alibaba/nacos/core/cluster/ServerMemberManager.java [hunk 1]
```diff
@@ -77,93 +78,93 @@
  */
 @Component(value = "serverMemberManager")
 public class ServerMemberManager implements ApplicationListener<WebServerInitializedEvent> {
-    
+
     private final NacosAsyncRestTemplate asyncRestTemplate = HttpClientBeanHolder
             .getNacosAsyncRestTemplate(Loggers.CORE);
-    
+
```
Comment (haoyann) [inline core/src/main/java/com/alibaba/nacos/core/cluster/ServerMemberManager.java:84]: Please fix code style, code style file in `/style`

Comment (cutiechi) [reply core/src/main/java/com/alibaba/nacos/core/cluster/ServerMemberManager.java:84]: Sorry, I'm going

Comment (cutiechi) [reply core/src/main/java/com/alibaba/nacos/core/cluster/ServerMemberManager.java]: @haoyann Please help me review again

File: core/src/main/java/com/alibaba/nacos/core/cluster/ServerMemberManager.java [hunk 2]
```diff
@@ -81,6 +82,11 @@
     private final NacosAsyncRestTemplate asyncRestTemplate = HttpClientBeanHolder
             .getNacosAsyncRestTemplate(Loggers.CORE);
     
+    /**
+     * Broadcast this node element information task.
+     */
+    private final MemberInfoReportTask infoReportTask = new MemberInfoReportTask();
```
Comment (haoyann) [inline core/src/main/java/com/alibaba/nacos/core/cluster/ServerMemberManager.java:88]: Please do not modify if there is no change.

File: core/src/main/java/com/alibaba/nacos/core/cluster/ServerMemberManager.java [hunk 3]
```diff
@@ -414,13 +415,13 @@ public void shutdown() throws NacosException {
     }
     
     @JustForTest
-    public void updateMember(Member member) {
-        serverList.put(member.getAddress(), member);
+    public void setMemberAddressInfos(Set<String> memberAddressInfos) {
```
Comment (haoyann) [inline core/src/main/java/com/alibaba/nacos/core/cluster/ServerMemberManager.java:418]: Same reason.

File: core/src/main/java/com/alibaba/nacos/core/cluster/ServerMemberManager.java (hunk 2 above)
Comment (cutiechi) [reply core/src/main/java/com/alibaba/nacos/core/cluster/ServerMemberManager.java:88]: Ok

File: core/src/main/java/com/alibaba/nacos/core/cluster/ServerMemberManager.java [hunk 4]
```diff
@@ -414,13 +415,13 @@ public void shutdown() throws NacosException {
...
+        this.memberAddressInfos = memberAddressInfos;
     }
     
     @JustForTest
-    public void setMemberAddressInfos(Set<String> memberAddressInfos) {
-        this.memberAddressInfos = memberAddressInfos;
+    public void updateMember(Member member) {
```
Comment (haoyann) [inline core/src/main/java/com/alibaba/nacos/core/cluster/ServerMemberManager.java:423]: Same reason.

File: core/src/main/java/com/alibaba/nacos/core/cluster/ServerMemberManager.java [hunk 5]
```diff
@@ -464,42 +465,40 @@ protected void executeBody() {
...
-                                                        ExceptionUtil.getAllExceptionMsg(throwable));
-                                        MemberUtil.onFail(ServerMemberManager.this, target, throwable);
-                                    }
-                                    
-                                    @Override
-                                    public void onCancel() {
-                                    
-                                    }
-                                });
+                        .post(url, header, Query.EMPTY, getSelf(), reference.getType(), new Callback<String>() {
```
Comment (haoyann) [inline core/src/main/java/com/alibaba/nacos/core/cluster/ServerMemberManager.java:471]: Same reason.

File: core/src/main/java/com/alibaba/nacos/core/cluster/ServerMemberManager.java (hunk 3 above)
Comment (cutiechi) [reply core/src/main/java/com/alibaba/nacos/core/cluster/ServerMemberManager.java:418]: Ok

File: core/src/main/java/com/alibaba/nacos/core/cluster/ServerMemberManager.java (hunk 4 above)
Comment (cutiechi) [reply core/src/main/java/com/alibaba/nacos/core/cluster/ServerMemberManager.java:423]: Ok

File: core/src/main/java/com/alibaba/nacos/core/cluster/ServerMemberManager.java (hunk 5 above)
Comment (cutiechi) [reply core/src/main/java/com/alibaba/nacos/core/cluster/ServerMemberManager.java:471]: Ok

Comment (haoyann) [inline core/src/main/java/com/alibaba/nacos/core/cluster/ServerMemberManager.java:471]: Same reason.

Comment (cutiechi) [reply core/src/main/java/com/alibaba/nacos/core/cluster/ServerMemberManager.java:471]: Ok

**Comment by cutiechi** [comment]
@haoyann Please help me review again

**Review by haoyann — APPROVED** [review]
LGTM

**Review by KomachiSion — APPROVED** [review]

**Pull request merged**